slashing-protection/src/main/java/tech/pegasys/web3signer/slashingprotection/DbSlashingProtection.java

```diff
@@ -231,25 +227,13 @@ public void registerValidators(final List<Bytes> validators) {
       return;
     }
 
-    jdbi.useTransaction(
-        SERIALIZABLE,
-        h -> {
-          final List<Validator> existingRegisteredValidators =
-              validatorsDao.retrieveValidators(h, validators);
-          final List<Bytes> existingValidatorsPublicKeys =
-              existingRegisteredValidators.stream()
-                  .map(Validator::getPublicKey)
-                  .collect(Collectors.toList());
-
-          final List<Bytes> validatorsMissingFromDb = new ArrayList<>(validators);
-          validatorsMissingFromDb.removeAll(existingValidatorsPublicKeys);
+    final List<Validator> insertedValidators =
```

Comment:
Does insertedValidators that is returned include the existing validators not just the inserted ones? Might be good to rename this variable if that's the case

Reply:
renamed variable to make the intent more clear.